model/trace/database.yaml

```diff
@@ -453,6 +454,28 @@ groups:
         examples: [ '"{\"query\":{\"term\":{\"user.id\":\"kimchy\"}}}"' ]
       - ref: server.address
       - ref: server.port
+      - id: cluster_name
```

Comment:
Do you know if there is/could be more attributes under `cluster`?. Since we have `name` and `instance` here, I wonder if this doesn't make more sense under a namespace?

Reply:
As far as I know, these will be the only two attributes we would expect to be able to collect and report from our instrumented language clients. I can't foresee additions, but I'm happy to drop them under a namespace to give us scope if that changes.

Reply:
It's always a tradeoff, but since both you and @AlexanderWert  are the subject matter experts in the product, I will defer this decision to you. You would need to consider the breaking changes and how those would be handled if you want to introduce the namespace later vs now.

Reply:
Thanks @joaopgrassi. After discussing it, it makes sense to future-proof this and namespace it from day one rather than be faced with difficult decisions in the future if another valid identifier is present that we want to recommend capturing. PR is updated to reflect that decision.